### ISSUE TITLE
active scan mode can not get connect_ind packet

### ISSUE THREAD

**Issue by LeeKoTo** [nccgroup/Sniffle#68]
active scan mode can not get connect_ind packet

**Comment by sultanqasim** [comment]
Correct, that's the intended behaviour. Active scanning transmits SCAN_REQ to get SCAN_RSP. It's not meant to capture CONNECT_IT. Use the normal `sniff_receiver.py` script to capture CONNECT_IND and follow connections,
